### ISSUE TITLE
Including pre-trained models?

### ISSUE THREAD

**Issue by JPFrancis** [AllenCellModeling/pytorch_fnet#140]
Hi, I work with a group imaging Pancreatic Beta Cell pathology at USC. Would you be open to including your pre-trained models? We use the same imaging set up, but don't quite have the volume to train. It would be interesting to see to what extent your structural models would transfer. Thanks

**Comment by fcollman** [comment]
Hi, I think having the same setup is only the start of potential problems you might have.  Same magnification, same adjustments to the illumination etc, might all affect results.  If it works, great, but if it doesn't I wouldn't know what to conclude, as many things might be different.  When you say you don't have the 'volume' what do you mean?  I think having a training workflow setup will enable you to do a lot of things.  We think you need only ~50 3d stacks with 20-30 slices on a 2000x2000 camera to get training to a reasonable place for most of the structures.

**Comment by JPFrancis** [comment]
Thanks for the response. Right now we have ~200 manually labeled slices total (as opposed 1000-1500) and the labels are for mitochondria only. I didn't think that was quite enough to train yet and a colleague was under the impression that the Cell Explorer project had pre-trained models that would take brightfield images as input and output predictions for some set of structures. Or at least he assumed that was the vision in some sense. No problem if that is not the case. We can run through our training pipelines as usual. I just thought this might be worth a shot given similar imaging conditions.

**Comment by fcollman** [comment]
I'll ask around if we can upload the pre-trained models somewhere, but all the data we used to train the models and the scripts for training them are available, so you can recreate them yourself.  You'll want to become familiar with the workflows for training and inference anyway if you want to use the models, so i'm not sure providing the pre-trained models will save you much time.

**Comment by JPFrancis** [comment]
Okay, fair enough. Thank you

**Comment by counkomol** [comment]
By the way, we have some plans (over the next few weeks) to provide pre-trained models for the current version of the platform. However, we only have models for a select few structures, and be aware that the models likely will not work well on your data. The pre-trained models might be a good starting point for transfer learning though.

**Comment by JPFrancis** [comment]
Awesome, okay. I'll look out for that. Thanks for following up.
